models/Ion_auth_model.php

```diff
@@ -182,6 +182,14 @@ public function __construct()
 		$this->load->helper('date');
 		$this->lang->load('ion_auth');
 
+		// PHP password_* function sanity check
+		if (!function_exists('password_hash') || !function_exists('password_verify'))
+		{
+			show_error("PHP function password_hash or password_verify not found. " .
+						"Are you using CI 2 and PHP < 5.5? " .
+						"Please upgrade to CI 3, or PHP >= 5.5, or use password_compat.");
```

Comment:
Throw a link to password_compat in this message please - or reference a link thats in the readme and add it there.  I'm sure a lot of people won't know what it is offhand.

Reply:
Yeah, I spent a lot of time wondering about it actually. I was hoping people would just google it. But you're right. I'll add the link to password_compat.